src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -61,7 +61,7 @@ export default function MentionSuggestionsPopover({
                 id={`${usersListboxId}-${u.username}`}
                 className={classnames(
                   'flex justify-between items-center gap-x-2',
-                  'rounded p-2 leading-4 hover:bg-grey-2',
+                  'rounded p-2 leading-tight hover:bg-grey-2',
```

Comment:
Can you add a comment here noting why the leading customization is needed, as the issue is not immediately obvious.